test/account_set-test.js

```diff
@@ -1,8 +1,9 @@
+let {alice,bob,carol,dan,root,mtgox,bitstamp,amazon} = require('./testconfig').accounts;
```

Comment:
I really dislike this notation for assigning such a large group of accounts. If anyone modifies `accounts`, except maybe adding to the end of the list, we now have to find every instance of this list and update it? Ugh.